release_notes.md

```diff
@@ -6,10 +6,6 @@
   * `Keanu.Sampling.SimulatedAnnealing`
   * `Keanu.Sampling.MCMC` (to automatically choose between `MetropolisHastings` and `NUTS`)
 * The `PosteriorSamplingAlgorithm` objects are now immutable: you cannot set their values after construction.
-* When you choose a custom configuration of `MetropolisHastings` or `SimulatedAnnealing`, you must specify:
```

Comment:
Can we leave this in but change it slightly. As people can still customise the proposal and rejection strategies.

`you can specify (defaults are prepared)... `

and: ```the proposal distribution - the default option is PriorProposalDistribution. Options are `PriorProposalDistribution` and `GaussianProposalDistribution```

```a proposal rejection strategy - the default option is RollBackToCachedValuesOnRejection. Options are RollBackToCachedValuesOnRejection and RollbackAndCascadeOnRejection```

Reply:
👍